c/tests/integration/driver_test.c

```diff
@@ -0,0 +1,443 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "c/typedb_driver.h"
+#include "common.h"
+
+// Avoid helpers so the tests serve as documentation for the C API.
+
+bool test_database_management() {
+    const char dbName[] = "test_database_management";
+
+    Connection* conn = 0;
+    DatabaseManager* dbMgr = 0;
```

Comment:
We should use full variable names. This is modern C, we don't have to save bytes anymore.